### PR TITLE
Message queue docs

### PR DESCRIPTION
**Pull request by ozgunozerk**
Related to #15 (after xcmp queue docs, the issue will be completely closed)

- [x] Documentation

### CODE REVIEW

**Comment by ozgunozerk** [comment]
Message queue does not changed between v1.3.0 and v1.5.0. 
So even if we decide for or against v1.3.0, it doesn't matter :)

File: docs/modules/ROOT/pages/pallets/message-queue.adoc [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+* `QueuePaused` -- if the queue that overweight message to be executed belongs to is paused.
+* `NoPage` -- if the page that overweight message to be executed belongs to does not exist.
+* `NoMessage` -- if the overweight message could not be found.
+* `Queued` -- if the overweight message is already scheduled for future execution.
+For a message to be labeled as overweight, the pallet must have previously attempted execution and
+encountered failure due to insufficient weight for processing. Once marked as overweight, the message
+is excluded from the queue for future execution. However, if the overweight message is unexpectedly
+found in the queue of pending items, it indicates an anomaly. In such instances, an error is raised,
+as the message should have been excluded from future execution attempts following its initial failure
+due to being overweight.
```
[the remark this answers is not in the document]
Comment (ozgunozerk) [reply docs/modules/ROOT/pages/pallets/message-queue.adoc:69]: i liked yours better! thanks

**Comment by KitHat** [comment]
Could you please add a link to `docs/modules/ROOT/nav.adoc`?

File: docs/modules/ROOT/pages/pallets/message-queue.adoc [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            3. we processed messages from queue 3,
+                1. but weight was insufficient for the next message in queue 3,
+                2. so we switched to queue 4, (we don’t bump the service head for that)
+                3. we processed a message from queue 4
+                4. weight was insufficient for queue 4 and other queues as well, and we made a round trip across queues, till we reach queue 3, and we stopped.
+            4. `service_queues` call finished
+            5. service head is on queue 3 (there are still messages in queue 3)
+            6. we called `service_queue` again, which bumped the service head to queue 4
+            7. we continue processing from queue 4, although we were processing queue 4 in the last call
+
```
Comment (KitHat) [inline docs/modules/ROOT/pages/pallets/message-queue.adoc:154]: Looks like something went wrong with the formatting there
It is just a plain wall of text: https://deploy-preview-58--docs-oz-polkadot.netlify.app/substrate-runtimes/0.1.0/pallets/message-queue#important_mentions_and_faqs

**Review by KitHat — CHANGES_REQUESTED** [review]
Content looks good to me, but we need to fix some formatting IMO

**Review by KitHat — APPROVED** [review]

**Pull request merged**